apps/activities/templates/a4_candy_activities/activity_detail.html

```diff
@@ -1,6 +1,8 @@
 {% extends extends %}
 {% load wagtailcore_tags ckeditor_tags %}
 
+// FIXME not used
```

Comment:
is there a reason not to remove it?

Reply:
No, not really. Please discuss in the corresponding issue: https://github.com/liqd/adhocracy-plus/issues/597